app/src/components/common/CardSignInRequired.vue

```diff
@@ -23,21 +23,6 @@
           title="Ledger Nano"
           route="/ledger"
         />
-        <LiSession
```

Comment:
@jbibla I still think we should add a note about the user menu here. Or have a call to action "Sign into an existing address" which closes the card and opens the user menu?